test/runtests.jl

```diff
@@ -15,6 +15,7 @@ using TimeZones
     include("power.jl")
     include("scaling.jl")
     include("temporal.jl")
+    include("transformers.jl")
```

Comment:
this might be a good opportunity to break out the interface stuff (`transform`, `Transform`) from all the `apply` methods into two files. Or at least rename `transformers.jl` to `transform.jl` (the original naming was a clash when the package was called `Transform.jl`)

What do you think?

Reply:
I'd be happy with separate `transform.jl` and `apply.jl`.